src/ontology/uberon-edit.obo

```diff
@@ -223621,6 +223621,15 @@ relationship: overlaps UBERON:0002625 ! median preoptic nucleus
 relationship: overlaps UBERON:0002690 ! anteroventral periventricular nucleus
 relationship: part_of UBERON:0006569 ! diencephalic nucleus
 
+[Term]
+id: UBERON:8470000
+name: placental blood
+def: "Blood located in the placenta." [https://www.ncbi.nlm.nih.gov/books/NBK53254/] {http://purl.org/dc/terms/contributor="http://orcid.org/0000-0001-6595-0902"}
```

Comment:
Add contributor anno straight to term, not on definitions.

Reply:
If you want to indicate that you invented that definition, use xref, not contributor:

```
def: "Blood located in the placenta." [https://www.ncbi.nlm.nih.gov/books/NBK53254/, http://orcid.org/0000-0001-6595-0902]
```